### PR TITLE
[Create timepoint] remove markTestSkipped

### PR DESCRIPTION
**Pull request by kongtiaowang**
This pull request removes markTestSkipped.

### CODE REVIEW

File: modules/create_timepoint/test/create_timepointTest.php [hunk 1]
```diff
@@ -162,16 +158,13 @@ private function _createTimepoint($canID, $subproject, $visitlabel)
...
-            'Skipping tests until create timepoint works well'
-        );
         $this->safeGet(
             $this->url . "/create_timepoint/?candID=900000&identifier=900000"
         );
         $this->webDriver->findElement(WebDriverBy::Name("fire_away"))->click();
         $bodyText = $this->webDriver->getPageSource();
-        $this->assertContains(
-            "A visit label is required for creating a timepoint.",
+        $this->assertNotContains(
```
Comment (driusan) [inline modules/create_timepoint/test/create_timepointTest.php]: I don't see why this change is required. Is the error message not being displayed or has the wording changed?

Comment (kongtiaowang) [inline modules/create_timepoint/test/create_timepointTest.php]: @driusan 
This error message has been changed.
It has been replaced by React drop-down list. It never shows.
